src/pages/Task.tsx

```diff
@@ -127,6 +125,16 @@ const TaskCore: React.FC = () => {
     taskStatus: task?.status,
   });
 
+  if (isPerfPluginEnabled) {
```

Comment:
when both plugins are enabled, the tabToIndexMap contains non-integer numbers and values repeat across the keys: `{"logs":0,"tests":1,"files":2,"trend-charts":3.5,"build-baron":3.5}`. I

Reply:
Whoops, nice catch! Should be fixed now and handle any combination of them being enabled